perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -131,8 +133,10 @@ def _PostCreate(self):
             instance = self.client.servers.get(self.id)
             status = instance.status
 
-        self.floating_ip = self.public_network.get_or_create()
-        instance.add_floating_ip(self.floating_ip)
+        with self._floating_ip_lock:
```

Comment:
This CL looks fine but is this a Workaround for an Openstack issue?  If so can you add a comment here with the "os" bug?